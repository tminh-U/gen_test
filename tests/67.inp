15
15 4 17 7 10 14 8 14 13 13 19 19 19 13 18 
10
2 9
9 13
6 12
10 10
4 13
12 13
11 15
9 14
10 13
15 15
