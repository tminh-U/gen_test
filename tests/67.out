54
32
48
2
304
4
11
36
16
2
